tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -154,23 +147,6 @@ def test_universal_autolog_calls_specific_autologs_correctly(library, mlflow_mod
         )
 
 
-def test_universal_autolog_calls_pyspark_immediately():
-    mlflow.autolog()
-    assert not autologging_is_disabled(mlflow.spark.FLAVOR_NAME)
-
-    mlflow.autolog(disable=True)
-    assert autologging_is_disabled(mlflow.spark.FLAVOR_NAME)
-
-    mlflow.autolog(disable=False)
-    assert not autologging_is_disabled(mlflow.spark.FLAVOR_NAME)
-
-    with mock.patch("mlflow.spark.autolog", wraps=mlflow.spark.autolog) as autolog_mock:
-        # there should be no import hook on pyspark since autologging was already
-        # applied to an active spark session
-        mlflow.utils.import_hooks.notify_module_loaded(pyspark)
-        autolog_mock.assert_not_called()
```

Comment:
Can we rename this test `test_universal_autolog_calls_pyspark_immediately_on_databricks` and modify the implementation?